<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidgetSong">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Song ID</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Song Name</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Singer</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>S1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Khong yeu dung gay thuong nho</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Minh Thanh</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>P2</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>Tinh tan</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Thanh Minh</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>P3</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Viet Nam toi oi</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>Hieu Hien</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>P4</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>Dap vo cay dan</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>Hieu Thanh</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>P5</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>Doi loi</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>Trung Thanh</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelSongName">
       <property name="text">
        <string>Song Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelSongID">
       <property name="text">
        <string>Song ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelSinger">
       <property name="text">
        <string>Singer:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditSongID"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditSongName"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditSinger"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
